packages/storefront-lwc/src/modules/commerce/basket/basket.html

```diff
@@ -84,14 +84,20 @@
             <div class="row">
                 <div class="col-md-8">
                     <template for:each={products} for:item="product">
-                        <commerce-product-line-item product={product} key={product.productId}>
+                        <commerce-product-line-item product={product} key={product.productId} onremovelineitem={updateBasket}>
```

Comment:
added this event handling after moving remove product mutation to this component.